practices/managing-static-assets.md

```diff
@@ -1,16 +1,13 @@
 # Managing static assets
 
-Static assets are files that don't need to be generated, modified or processed by an application before being delivered to the user. This typically includes CSS, JavaScript, images and web fonts.
+Static assets are files that don't need to be generated, modified or processed by an application before being delivered to the user. This typically includes CSS, JavaScript, images and web fonts. Static assets change rarely, and their content doesn't depend on user input or preferences.
 
-This content rarely changes and its contents don't depend on user input or preferences.
+We host on our own infrastructure these kinds of static resources, instead of including them on our pages from an third-party CDN. This applies especially to JavaScript files or libraries, and font files.
```

Comment:
"We host these kinds of static resources on our own infrastructure" instead of "We host on our own infrastructure these kinds of static resources"?